tests/docs/partials/too_large/test_payload_too_large.rst

```diff
@@ -0,0 +1,11 @@
+Scenario: An API consumer submitting a request with a request body containing 45,000 messages between 6MB and 10MB receives a 413 status code and response body containing an instance of that error
```

Comment:
I noticed that if I submitted a request larger than 10MB our code never gets a chance to enrich the error because apigee won't accept it, so this helps anyone working with the tests to understand what our tests cover and what they don't.

Reply:
Ok, do you think it's worthwhile us having a test for this 10MB limit? I'm happy for that to be the scope of a different ticket

Reply:
That's apigee behaviour that I don't think we can change. Given that it's got nothing to do with what we implement on this team I don't think we should be testing it.